config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1265 @@
+    url: "developers/integrations/"
+    parent: "dev_tools"
+    identifier: "dev_tools_integrations"
+    weight: 7
+  - name: "Create a new Integration"
+    url: "developers/integrations/new_check_howto/"
+    parent: "dev_tools_integrations"
+    weight: 701
+  - name: "Python"
+    url: "developers/integrations/python/"
+    parent: "dev_tools_integrations"
+    weight: 702
+  - name: "Legacy"
+    url: "developers/integrations/legacy/"
+    parent: "dev_tools_integrations"
+    weight: 703
+
+  - name: "Guides"
+    url: "developers/guide/"
+    parent: "dev_tools"
+    weight: 8
+
+#########
+## API ##
+#########
+
+  - name: "API"
+    url: "api/"
+    pre: "nav_api"
+    identifier: "api"
+    weight: 120000
+
+########################
+## ACCOUNT MANAGEMENT ##
+########################
+
+  - name: "Account Management"
+    url: "account_management/"
+    pre: "nav_account"
+    identifier: "account_management"
+    weight: 130000
+
+  - name: "Team management"
+    url: "account_management/team/"
+    parent: "account_management"
+    weight: 1
+
+  - name: "Organization settings"
+    url: "account_management/org_settings/"
+    parent: "account_management"
+    weight: 2
+
+  ## ACCOUNT MANAGEMENT - SAML
+
+  - name: "SSO with SAML"
+    url: "account_management/saml/"
+    identifier: "account_management_saml"
+    parent: "account_management"
+    weight: 3
+  - name: "Active Directory"
+    url: "account_management/saml/activedirectory/"
+    parent: "account_management_saml"
+    weight: 301
+  - name: "Auth0"
+    url: "account_management/saml/auth0/"
+    parent: "account_management_saml"
+    weight: 302
+  - name: "Azure"
+    url: "account_management/saml/azure/"
+    parent: "account_management_saml"
+    weight: 303
+  - name: "Google"
+    url: "account_management/saml/google/"
+    parent: "account_management_saml"
+    weight: 304
+  - name: "NoPassword"
+    url: "account_management/saml/nopassword/"
+    parent: "account_management_saml"
+    weight: 305
+  - name: "Okta"
+    url: "account_management/saml/okta/"
+    parent: "account_management_saml"
+    weight: 306
+  - name: "SafeNet"
+    url: "account_management/saml/safenet/"
+    parent: "account_management_saml"
+    weight: 307
+
+  - name: "Multi-org accounts"
+    url: "account_management/multi_organization/"
+    parent: "account_management"
+    weight: 4
+
+  - name: "Switching between orgs"
+    url: "account_management/org_switching/"
+    parent: "account_management"
+    weight: 5
+
+  - name: "Billing"
+    url: "account_management/billing/"
+    parent: "account_management"
+    weight: 6
+
+  - name: "API and Application Keys"
+    url: "account_management/api-app-keys/"
+    parent: "account_management"
+    weight: 7
+
+##############
+## SECURITY ##
+##############
+
+  - name: "Security"
+    url: "security/"
+    pre: "nav_security"
+    identifier: "security"
+    weight: 140000
+
+  - name: "Agent"
+    url: "security/agent/"
```

Comment:
This link jumps up to the Agent section while all the other links stay in the security section.

On the current site, the nav stays in the Security section.

Reply:
fixed